resources/lua/ttype.lua

```diff
@@ -1,16 +1,51 @@
 core.enable_protocol(24)
+
 local program, _ = blight.version()
 local term = os.getenv("TERM") or "xterm-256color"
-local mtts = "MTTS 271"
 
 local index = 1
 local mod = {}
+local auto_reader_mode = true
+
+TTYPE_OPT_MTTS_ANSI =   0x001 -- Ansi support
+TTYPE_OPT_MTTS_VT100 =  0x002 -- VT100 support
+TTYPE_OPT_MTTS_UTF8 =   0x004 -- UTF-8 Support
+TTYPE_OPT_MTTS_256C =   0x008 -- 256 color support
+TTYPE_OPT_MTTS_MTRA =   0x010 -- Mouse tracking support
+TTYPE_OPT_MTTS_OSCC =   0x020 -- OSC color palette support (true color)
+TTYPE_OPT_MTTS_READ =   0x040 -- Client using screen reader
+TTYPE_OPT_MTTS_PROX =   0x080 -- This is a proxy connection
+TTYPE_OPT_MTTS_TRUC =   0x100 -- True color support
+TTYPE_OPT_MTTS_MNES =   0x200 -- Mud New Env Standard enabled
+TTYPE_OPT_MTTS_MSLP =   0x400 -- Mud Server Link Protocol enabled
```

Comment:
I'm honestly not a huge fan of this. The length of these constants is silly and I don't think they should be global.

Personally, I would strip the `TTYPE_OPT` prefix, and make these a member of the module.

I also think some of them need renaming. `PROX` and `READ` should be `PROXY` and `READER` or maybe even `SCREEN_READER`

Reply:
I work with C and low level kernel code now. Can you tell by my naming habits?

I was thinking that people might want to use the variables in scripts. But let's just make them local instead.

Reply:
With the current API, users do need access to these. (For `add_option` and `rem_option`)

Making these local and breaking `add_option` and `rem_option` apart into a function for each option is probably the best solution. What I actually mean, though, was making these constants part of the module rather than making them global or local. (`mod.MTTS_PROXY = 0x080`)

Reply:
I'll move them to part of the module. Of course. My lua has gotten rusty...

Reply:
As for "one function per setting". It feels like more work then reward.